aiofreepybox/api/player.py

```diff
@@ -0,0 +1,141 @@
+class Player:
+
+    def __init__(self, access):
+        self._access = access
+        self._api_version_target = 'v6'
+
+    player_volume_data_schema = {
+        'mute': bool(),
+        'volume': int
+    }
+
+    player_url_data_schema = {
+        'url': ''
+    }
+
+    media_control_seek_args = {
+        'seek_position': int,
+        'type': 'seek_position'
+    }
+
+    media_control_stream = {
+        'quality': '',
+        'source': ''
+    }
+
+    media_control_stream_args = {
+        'stream': media_control_stream,
+        'type': 'stream'
+    }
+
+    media_control_track_args = {
+        'track_id': int,
+        'type': 'track_id'
+    }
+
+    media_control_command = [
+        'play',
+        'pause',
+        'play_pause',
+        'stop',
+        'next',
+        'prev',
+        'seek_forward',
+        'seek_backward',
+        'seek_to',
+        'repeat_all',
+        'repeat_one',
+        'repeat_off',
+        'repeat_toggle',
+        'shuffle_on',
+        'shuffle_off',
+        'shuffle_toggle',
+        'record',
+        'record_stop',
+        'select_audio_track',
+        'select_srt_track',
+        'select_stream'
+    ]
+
+    media_control_data_schema = {
+        'args': media_control_stream_args,
+        'cmd': media_control_command[0]
+    }
+
+    async def get_player_status(self, player_id, api_version=None):
```

Comment:
I'm not sure it is necessary to accept an `api_version` for each call, I would specify it when constructing the Player instance and always use the same one afterwards. I don't think it makes sense for a client application to use multiple api versions concurrently.

Reply:
should be fixed in 45c4809